lib/util/sdp/index.js

```diff
@@ -510,15 +510,70 @@ function removeSSRCAttributes(sdp, ssrcAttributesToRemove) {
   ).join('\r\n');
 }
 
+/**
+ * Disable RTX in a given sdp.
+ * @param {string} sdp
+ * @returns {string} sdp without RTX
+ */
+function disableRtx(sdp) {
+  const mediaSections = getMediaSections(sdp);
+  const session = sdp.split('\r\nm=')[0];
+  return [session].concat(mediaSections.map(mediaSection => {
+    // Do nothing if the m= section does not represent a video track.
+    if (!/^m=video/.test(mediaSection)) {
+      return mediaSection;
+    }
+
+    // Create a map of codecs to payload types.
+    const codecsToPts = createCodecMapForMediaSection(mediaSection);
+    // Get the RTX payload types.
+    const rtxPts = codecsToPts.get('rtx');
+
+    // Do nothing if there are no RTX payload types.
+    if (!rtxPts) {
+      return mediaSection;
+    }
+
+    // Remove the RTX payload types.
+    const pts = new Set(getPayloadTypesInMediaSection(mediaSection));
+    rtxPts.forEach(rtxPt => pts.delete(rtxPt));
+
+    // Get the RTX SSRC.
+    const rtxSSRCMatches = mediaSection.match(/a=ssrc-group:FID [0-9]+ ([0-9]+)/);
+    const rtxSSRC = rtxSSRCMatches && rtxSSRCMatches[1];
+
+    // Remove the following lines associated with the RTX payload types:
+    // 1. "a=fmtp:<rtxPt> apt=<pt>"
+    // 2. "a=rtpmap:<rtxPt> rtx/..."
+    // 3. "a=ssrc:<rtxSSRC> cname:..."
+    // 4. "a=ssrc-group:FID <SSRC> <rtxSSRC>"
+    const filterRegexes = [
+      /^a=fmtp:.+ apt=.+$/,
```

Comment:
I am not super familiar with sdps, and wondering if these regex's could ever match unintended lines?
some of the expressions has "rtx" in them but some not.

Reply:
@makarandp0 I'll explain the SDP lines not having "rtx":

1. `/^a=fmtp:.+ apt=.+$/` - Only RTX fmtp lines have the associated payload type `apt`.
2. `/^a=ssrc-group:.+$/` - This line maps the primary SSRC to its RTX SSRC.

So, the lines matching the above regexes can be removed.